src/ArrayValue.php

```diff
@@ -128,6 +139,11 @@ public function join(ArrayValue $other);
      */
     public function slice(int $offset, int $length);
 
+    /**
+     * @return ArrayValue
+     */
+    public function splice(int $offset, int $length, array $replacement = []);
```

Comment:
`$replacement` should not be `ArrayValue` instead of simple `array` in this case?

Reply:
I guess it will be more consistent :+1: